src/main.js

```diff
@@ -50,53 +42,57 @@ function resetGame() {
   }
 }
 
-function startClassicGame() {
+const startClassicGame = () => {
   show([changeGameButton, allFighterIconsSection]);
   hide([classicGamePlayButton, difficultGameButton, broomFighterButton, hatFighterButton]);
   newGame.type = 'Classic';
   chooseYourGameTitleLine.innerText = "Choose Your Fighter!";
 }
 
-function startDifficultGame() {
+const startDifficultGame = () => {
   newGame.type = 'Difficult'
   show([changeGameButton, allFighterIconsSection, broomFighterButton, hatFighterButton]);
   hide([classicGamePlayButton, difficultGameButton]);
   chooseYourGameTitleLine.innerText = "Choose Your Fighter!";
 }
-function showChosenIcons() {
+const showChosenIcons = () => {
   show([playersChosenFighters]);
   playersChosenFighters.innerHTML =
   `<img class="player-icon-size human-chosen-icon-border" src="./assets/magic-${newGame.player.choice}.png">
   <img class="player-icon-size comp-chosen-icon-border" src="./assets/magic-${newGame.computer.choice}.png">`
 }
 
-function updateWinCount() {
+const updateWinCount = () => {
   humanWinCount.innerText = newGame.player.retrieveWinsFromStorage() || 0;
   compWinCount.innerText = newGame.computer.retrieveWinsFromStorage() || 0;
 }
 
-function updateWinText() {
+const updateWinText = () => {
   if (newGame.winner === 'Human' || newGame.winner === 'Computer') {
     chooseYourGameTitleLine.innerText = `${newGame.winner} wins!`
   } else {
     chooseYourGameTitleLine.innerText = `It's a draw!`
   }
 }
 
-function changeGame() {
+const changeGame = () => {
   show([classicGamePlayButton, difficultGamePlayButton]);
   hide([changeGameButton, allFighterIconsSection, playersChosenFighters]);
   chooseYourGameTitleLine.innerText = "Choose Your Game!";
 }
 
-function show(elements) {
-  for (var i = 0; i < elements.length; i++) {
-    elements[i].classList.remove('hidden');
-  }
+const show = (elements) => {
```

Comment:
Great job re-writing these functions to use both the arrow syntax, and the forEach iterator method.